cmd/bad-key-revoker/main.go

```diff
@@ -471,19 +477,85 @@ func main() {
 		logger:          logger,
 		clk:             clk,
 	}
+
+	// If the MaxBackoff.Duration was not set via the config, set it now
+	// so that it can't increase so high that it will not try on a
+	// reasonable timeframe.
+	if config.BadKeyRevoker.MaxBackoff.Duration == 0 {
+		config.BadKeyRevoker.MaxBackoff.Duration = time.Second * 60
+	}
+
+	// Set the backup policy
+	backoff := &BackoffPolicy{
+		Limit:   config.BadKeyRevoker.MaxBackoff.Duration,
+		Minimum: config.BadKeyRevoker.Interval.Duration,
```

Comment:
I think there's a bit of confusing naming going on here. The names "Limit" and "Minimum" aren't really a matched pair -- maybe "Limit" and "Initial", or "Maximum" and "Minimum". Similarly, the names "MaxBackoff" and "Interval" aren't really a matched pair. I understand preserving the name "Interval" since that already exists in the configs, but then maybe that suggests the new config parameter should be named "MaxInterval", to make it clear that they're a pair.

Reply:
I stopped using this struct, but did take the advice with the new variables and prefixed them with backoff* to help the reader understand what goes together.